src/core/meshcop/joiner.hpp

```diff
@@ -76,7 +76,9 @@ class Joiner : public InstanceLocator
      * @param[in]  aCallback         A pointer to a function that is called when the join operation completes.
      * @param[in]  aContext          A pointer to application-specific context.
      *
-     * @retval OT_ERROR_NONE  Successfully started the Joiner service.
+     * @retval OT_ERROR_BUSY          The previous attempt is still on-going.
+     * @retval OT_ERROR_INVALID_STATE The IPv6 stack is not enabled or Thread stack is fully enabled.
+     * @retval OT_ERROR_NONE          Successfully started the Joiner service.
```

Comment:
Just a style question, are you sorting these?
I think having success case `OT_ERROR_NONE` as the first on the list reads better.

Reply:
@bukepo did you intend to change this (move NONE case as first) or keep it as is? I think you changed the `joiner.h` one? Just checking since it's marked as resolved.

Reply:
@bukepo, see @abtink's question above.

Reply:
Sorry, I ignored the question. I will update the PR today.

Reply:
updated.